.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "upstream"]
 	path = upstream
-	url = http://sourceware.org/git/binutils-gdb.git
+	url = https://github.com/sailfishos-mirror/gdb.git
```

Comment:
@keto hmm we would have done better naming the mirror "binutils-gdb" - worth to care now?